src/rdkafka_cgrp.c

```diff
@@ -2658,6 +2940,11 @@ rd_kafka_cgrp_assign (rd_kafka_cgrp_t *rkcg,
         /* If the consumer has raised a fatal error we treat all
          * assigns as unassigns */
         if (rd_kafka_fatal_error_code(rkcg->rkcg_rk)) {
+                if (assignment)
+                        rd_kafka_dbg(rkcg->rkcg_rk, CGRP|RD_KAFKA_DBG_CONSUMER,
+                                     "ASSIGN", "Group \"%s\": Consumer is in "
+                                     "a failed state, treating assign as "
```

Comment:
"failed state" introduces new wording for an existing concept, please have it read "Consumer is in a fatal state/raised a fatal error/or something that says fatal and not failed".
The same goes for previous occurences.

Reply:
this is me being dyslexic.

Reply:
not addressed

Reply:
on the upside, i addressed it in two other places instead.